lib/Pipfile

```diff
@@ -22,7 +22,7 @@ verify_ssl = true
 black = ">=22.3.0"
 pre-commit = "*"
 hypothesis = ">=6.17.4"
-mypy = ">=0.930"
+mypy = ">=0.981"
```

Comment:
This new `mypy_main` syntax is a breaking change, so we no longer support any mypys below this one